app/experimenter/experiments/models/nimbus.py

```diff
@@ -294,14 +294,14 @@ def targeting(self):
         if self.locales.count():
             locales = [locale.code for locale in self.locales.all().order_by("code")]
 
-            if self.application == self.Application.DESKTOP:
-                expressions.append(f"locale in {locales}")
-            else:
-                iso_locales = {locale[:2] for locale in locales}
-                iso_locales_expression = " || ".join(
-                    [f"'{language}' in locale" for language in sorted(iso_locales)]
-                )
-                expressions.append(iso_locales_expression)
+            expressions.append(f"locale in {locales}")
```

Comment:
Sorry @yashika-khurana I just realized if we take this out now then in the time between landing this and landing the UI for languages mobile experiments won't work properly, let's split this into two.  We can add the language part below here and leave the language code untouched, and then file a follow up to remove the special case for mobile locales after we add the languages UI.

Reply:
Got it, but we do have UI ready as well https://github.com/mozilla/experimenter/pull/7269. Am I missing something? 🤔

Reply:
Yes makes sense, thank you for explaining it 😊 🙏